backend/FwLite/FwLiteShared/Projects/CombinedProjectsService.cs

```diff
@@ -77,8 +80,10 @@ public async ValueTask<IReadOnlyCollection<ProjectModel>> LocalProjects()
         await crdtProjectsService.EnsureProjectDataCacheIsLoaded();
         var crdtProjects = crdtProjectsService.ListProjects();
         //todo get project Id and use that to specify the Id in the model. Also pull out server
-        var projects = crdtProjects.ToDictionary(p => p.Name,
-            p => new ProjectModel(p.Name,
+        var projects = crdtProjects.ToDictionary(p => p.Name, // actually the code
```

Comment:
what if we rename `IProjectIdentifier.Name`? I'm just not sure what to rename it to? We could do Code, but it doesn't strictly match code either. We could call it ID and it's just a string.

Reply:
Yeah, I suppose ID actually makes pretty good sense 🤔. What if we called it `Identifier`? 😆. I know that's the same thing theoretically, but maybe it has a less rigid feel to it.